Cargo.toml

```diff
@@ -8,6 +8,21 @@ members = [
 	"chains/orchestrator-relays/node/*",
 	"chains/orchestrator-relays/runtime/dancelight",
 	"chains/orchestrator-relays/runtime/dancelight/constants",
+	"chains/xcm-integration-tests/emulated/chains/orchestrator-paras/dancebox",
```

Comment:
chains/xcm-integration-tests/*

Reply:
We can't exclude all of them, because we need to either specify the path to the folder where we have toml file, or glob: `*` for top level, but the exact next level should have toml files. So, I've removed some and replaced with global, where it possible.